### PR TITLE
toucahud

### PR DESCRIPTION
**Pull request by toucaknight**
made a new hud and wanted to post it on here!

https://tf2huds.dev/hud/toucaHUD

posted it on here before getting to this one

### CODE REVIEW

File: hud-data/toucahud.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "Jofre",
+    "Hypnotize",
+    "Colly",
+    "Omnibombulator"
+  ],
+  "releaseDate": "2024-10-03",
+  "parent": "hexhud, collyhud",
+  "distinguishedFromParent": true,
+  "resources": [
+    "https://imgur.com/2CbDLfM"
```
Comment (mastercoms) [inline hud-data/toucahud.json:14]: This should instead by a list of the images you posted in hud-resources.

File: hud-data/toucahud.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  ],
+  "flags": [
+    "fonts",
+    "menus",
+    "scoreboards",
+    "minmode",
+    "materials"
+  ],
+  "tags": [
+    "Minimal",
```
Comment (mastercoms) [inline hud-data/toucahud.json:24]: Please lower-case tags.

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: hud-data/toucahud.json (hunk 2 above)
Comment (toucaknight) [reply hud-data/toucahud.json]: made all the changes, lemme know if there's anything else you need me to do

File: hud-data/toucahud.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    "Jofre",
+    "Hypnotize",
+    "Colly",
+    "Omnibombulator"
+  ],
+  "releaseDate": "2024-10-03",
+  "parent": "hexhud",
+  "distinguishedFromParent": true,
+  "resources": [
+    "toucahud-banner, toucahud-menu, toucahud-backpackui, toucahud-backpackheavy, toucahud-ingameui, toucahud-ingamemeter, toucahud-ingamestickycount, toucahud-ingameheadcount, toucahud-ingameubercharge, toucahud-ingamenobuilding, toucahud-ingamebuildmenu, toucahud-ingameallbuilding, toucahud-ingamedestroybuild, toucahud-ingamedisguise, toucahud-ingametauntmenu"
```
Comment (mastercoms) [inline hud-data/toucahud.json:14]: Do it like this:

"toucahud-banner",
"toucahud-menu"

File: hud-data/toucahud.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      "tc"
+    ],
+    "res": [
+      "16/9"
+    ],
+    "position": [
+      "center"
+    ],
+    "lang": [
+      "English"
```
Comment (mastercoms) [inline hud-data/toucahud.json:68]: Should be "en"

**Review by mastercoms — CHANGES_REQUESTED** [review]

**Comment by toucaknight** [comment]
i updated the hash and fixed the remaining issues

**Comment by toucaknight** [comment]
Any way I could fix the check HUD error, idk what "Error: Process completed with exit code 1" exactly is. still new to all of this

**Comment by toucaknight** [comment]
i noticed right under my hud's .json file, it had a string talking about discord tags.
image

**Comment by mastercoms** [comment]
Should be .touca#0

**Comment by toucaknight** [comment]
did the discord tag correctly

**Comment by toucaknight** [comment]
Sweet! all the checks passed! is there anything else I need to do?

**Comment by toucaknight** [comment]
i just made the branch up to date

**Review by mastercoms — APPROVED** [review]

**Pull request merged**